go.mod

```diff
@@ -2,9 +2,49 @@ module github.com/datawire/ambassador
 
 go 1.13
 
-// `go mod tidy` is going to try to remove `github.com/cncf/udpa`--don't let it!
-// That will break `make generate`.  Keep the github.com/cncf/udpa version
-// in-sync with the github.com/cncf/udpa/go version.
+// If you're editing this file, there's a few things you should know:
```

Comment:
Shouldn't we keep that comment about `go mod tidy`? Or is it no longer relevant?

Reply:
@LukeShu any opinion on this?

Reply:
Good question

Reply:
Yes, the comment is still applicable.  It seems that it was dropped in the 8918d359dc51d5b7ead5fbae93620f51d7ba1b97 merge.

Reply:
No, I tried that; it didn't work right :shrug: